wrappers/python/tests/pool/test_open_pool_ledger.py

```diff
@@ -1,58 +1,81 @@
-    pool_handle = await open_pool_ledger("pool_1", "")
+    await pool.create_pool_ledger_config(
+        "pool_1",
+        json.dumps({
+            "genesis_txn": str(pool_utils.create_genesis_txn_file_for_test_pool("pool_1"))
+        }))
+
+    pool_handle = await pool.open_pool_ledger("pool_1", None)
     assert pool_handle is not None
+
     await pool.close_pool_ledger(pool_handle)
 
 
 @pytest.mark.asyncio
 async def test_open_pool_ledger_works_for_config(cleanup_storage):
-    config = '{"refreshOnOpen": true}'
-    await pool.create_pool_ledger_config("pool_1")
-    pool_handle = await open_pool_ledger("pool_1", config)
+    await pool.create_pool_ledger_config(
+        "pool_1",
+        json.dumps({
+            "genesis_txn": str(pool_utils.create_genesis_txn_file_for_test_pool("pool_1"))
+        }))
+
+    pool_handle = await pool.open_pool_ledger("pool_1", '{"refreshOnOpen": true}')
     assert pool_handle is not None
+
     await pool.close_pool_ledger(pool_handle)
 
 
 @pytest.mark.asyncio
 async def test_open_pool_ledger_works_for_twice(cleanup_storage):
-    await pool.create_pool_ledger_config("pool_1")
-    pool_handle = await open_pool_ledger("pool_1", "")
+    await pool.create_pool_ledger_config(
+        "pool_1",
+        json.dumps({
+            "genesis_txn": str(pool_utils.create_genesis_txn_file_for_test_pool("pool_1"))
+        }))
+
+    pool_handle = await pool.open_pool_ledger("pool_1", None)
     assert pool_handle is not None
+
     with pytest.raises(IndyError) as e:
-        await open_pool_ledger("pool_1", "")
+        await pool.open_pool_ledger("pool_1", "")
+
     assert ErrorCode.PoolLedgerInvalidPoolHandle == e.value.error_code
     await pool.close_pool_ledger(pool_handle)
 
 
 @pytest.mark.asyncio
 async def test_open_pool_ledger_works_for_two_nodes(cleanup_storage):
-    nodes = [
-        "{\"data\":{\"alias\":\"Node1\",\"client_ip\":\"10.0.0.2\",\"client_port\":9702,\"node_ip\":\"10.0.0.2\",\"node_port\":9701,\"services\":[\"VALIDATOR\"]},\"dest\":\"Gw6pDLhcBcoQesN72qfotTgFa7cbuqZpkX3Xo6pLhPhv\",\"identifier\":\"Th7MpTaRZVRYnPiabds81Y\",\"txnId\":\"fea82e10e894419fe2bea7d96296a6d46f50f93f9eeda954ec461b2ed2950b62\",\"type\":\"0\"}\n",
-        "{\"data\":{\"alias\":\"Node2\",\"client_ip\":\"10.0.0.2\",\"client_port\":9704,\"node_ip\":\"10.0.0.2\",\"node_port\":9703,\"services\":[\"VALIDATOR\"]},\"dest\":\"8ECVSk179mjsjKRLWiQtssMLgp6EPhWXtaYyStWPSGAb\",\"identifier\":\"EbP4aYNeTHL6q385GuVpRV\",\"txnId\":\"1ac8aece2a18ced660fef8694b61aac3af08ba875ce3026a160acbc3a3af35fc\",\"type\":\"0\"}\n"
-    ]
-    await pool.create_pool_ledger_config("pool_1", nodes)
-    pool_handle = await open_pool_ledger("pool_1", None)
+    await pool.create_pool_ledger_config(
+        "pool_1",
+        json.dumps({
+            "genesis_txn": str(pool_utils.create_genesis_txn_file_for_test_pool("pool_1", 2))
+        }))
+
+    pool_handle = await pool.open_pool_ledger("pool_1", None)
```

Comment:
See my comment about create_pool_ledger_config. We have an exactly same situation here.